pandas/core/series.py

```diff
@@ -419,14 +419,30 @@ def dtypes(self):
     def ftype(self):
         """
         Return if the data is sparse|dense.
+
+        .. deprecated:: 0.25.0
+           Use :func:`dtype` instead.
```

Comment:
@datapythonista  will these render?

Reply:
yes, I think this is correct and should render as a box like the one in http://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.read_table.html